src/server-code/admin-server.js

```diff
@@ -19,8 +19,10 @@ import { exec } from 'child_process';
 // Helps us parse post requests from falcor
 import bodyParser from 'body-parser';
 
+import request from 'request';
+
 /* Our own helper functions */
-import { isDevelopment, filterByEnvironment, hash } from 'lib/utilities';
+import { isDevelopment, filterByEnvironment, hash, googleClient, googleWhitelist } from 'lib/utilities'; // eslint-disable-line max-len
```

Comment:
You don't need to keep disabling all these max line length linting errors, feel free to refactor it to
```js
import {
  isDevelopment,
  filterByEnvironment,
  hash,
  googleClient,
  googleWhitelist,
} from 'lib/utilities';
```